src/pages/NewChatPage.tsx

```diff
@@ -113,37 +112,22 @@ function NewChatPage({isGroupChat}: NewChatPageProps) {
     const {insets} = useStyledSafeAreaInsets();
     const [isSearchingForReports] = useOnyx(ONYXKEYS.IS_SEARCHING_FOR_REPORTS, {initWithStoredValues: false});
 
-    const {
-        headerMessage,
-        maxParticipantsReached,
-        searchTerm,
-        debouncedSearchTerm,
-        setSearchTerm,
-        selectedOptions,
-        setSelectedOptions,
-        recentReports,
-        personalDetails,
-        userToInvite,
-        areOptionsInitialized,
-    } = useOptions({
-        isGroupChat,
-    });
+    const {headerMessage, searchTerm, debouncedSearchTerm, setSearchTerm, selectedOptions, setSelectedOptions, recentReports, personalDetails, userToInvite, areOptionsInitialized} =
+        useOptions({
+            isGroupChat,
+        });
 
     const [sections, firstKeyForList] = useMemo(() => {
         const sectionsList: OptionsListUtils.CategorySection[] = [];
         let firstKey = '';
 
-        const formatResults = OptionsListUtils.formatSectionsFromSearchTerm(debouncedSearchTerm, selectedOptions, recentReports, personalDetails, maxParticipantsReached);
+        const formatResults = OptionsListUtils.formatSectionsFromSearchTerm(debouncedSearchTerm, selectedOptions, recentReports, personalDetails, false);
```

Comment:
```suggestion
        const formatResults = OptionsListUtils.formatSectionsFromSearchTerm(debouncedSearchTerm, selectedOptions, recentReports, personalDetails);
```

Reply:
I think the default value for parameter is false, so no need to pass it.

Reply:
@GandalfGwaihir Ah, you are right, it is not having default value, ignore this change it is not affected.

Reply:
Why do we still need this param? Is there some case where `maxOptionsSelected` applies anymore?

Reply:
Is there any place left in code where we should pass `maxOptionsSelected: true` to `formatSectionsFromSearchTerm()`?

Reply:
Yes, it used in money request here.
https://github.com/Expensify/App/blob/64695c850dfd7dc054823b69f35c9511cd934cdd/src/pages/iou/request/MoneyTemporaryForRefactorRequestParticipantsSelector.js#L136

Reply:
Hmm, why should there be a participant limit on a money request...? If you have multiple participants then you are splitting which creates a Group Chat. I think we can remove the restriction there too.

Reply:
Reviewing today.

Reply:
>Also, do you know why there was a limit on Money request participant selector in the first place?

Sure! And sorry, I should have caught this sooner. But it's because the "split" request lands on a Group DM between you and all the other participants. Since we had a previous restriction on that. Now that there is none for Group Chats we can do the same behavior everywhere.